src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: $card-width;
+  background-color: $card-bg-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: transform 300ms;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    margin-top: 40px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: $prod-name-size;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: $prod-code-size;
+    line-height: 14px;
+    margin-top: 4px;
+    color: $second-text-color;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews-numb {
+    font-weight: 400;
+    font-size: $reviews-text-size;
+    line-height: 14px;
+    color: $main-text-color;
+  }
+
+  &__prices {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: $price-text-size;
+    line-height: 18px;
+    color: $second-text-color;
+  }
+
+  &__amount {
+    font-weight: 700;
+    font-size: $price-amount-size;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 16px auto auto;
+
+    width: $button-width;
+    height: $button-height;
```

Comment:
it's not a good practice to hardcode element with content inside. Just use padding